en/scripting/ccclass.md

```diff
@@ -0,0 +1,484 @@
+## ccclass
+
+When the decorator [ccclass](#ccclass) is applied to a class, the class is called `ccclass`. The `ccclass` injects additional information to control __Cocos Creator 3D__'s serialization of this kind of object along with the editor's display of this kind of object, etc.
```

Comment:
to control the serialization and editor interoperation of this type of objects.

Reply:
I can’t use that exactly. It’s not correct. I can say:

these types of objects. 

Or 

this type of object. 

Which would you prefer?

Reply:
I ended up using this:

editor's display for these types of objects.